docs/bokeh/make.bat

```diff
@@ -29,6 +29,7 @@ if "%1" == "help" (
 if "%1" == "clean" (
 	rmdir %BUILDDIR% /s /q
 	del /q /s source\docs\gallery\*
+	del /q /s source\docs\examples\*
```

Comment:
@bryevdv This is one change I carried over from the Linux Makefile, but I think more updates to make.bat are required. Is that something you could look into? I could also make a separate issue about that!

Reply:
I guess just make an issue, I don't really use windows with any regularity and am not set up to do any dev on windows.